buildscripts/gateway-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -e
+
+export SERVER_ENDPOINT=127.0.0.1:24240
+export ENABLE_HTTPS=0
+export ACCESS_KEY=minio
+export SECRET_KEY=minio123
+export MINIO_ACCESS_KEY=minio
+export MINIO_SECRET_KEY=minio123
+export AWS_ACCESS_KEY_ID=minio
+export AWS_SECRET_ACCESS_KEY=minio123
+
```

Comment:
As a proper bash script can you convert these to functions? and move the ENVs into `function init()` ?